src/test/java/SetTest.java

```diff
@@ -0,0 +1,46 @@
+import java.util.HashSet;
+import java.util.Set;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("세트의 사이즈는 3이 되어야 한다.")
+    void checkSetSize() {
+        Assertions.assertThat(numbers).hasSize(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

import static 을 사용하면 코드의 가독성이 더 좋아질 것 같습니다.😀
`import static org.assertj.core.api.Assertions.assertThat;`